diff -u -r ij.orig/source/ij/gui/PlotWindow.java ij/source/ij/gui/PlotWindow.java
--- ij.orig/source/ij/gui/PlotWindow.java	2014-10-18 15:10:32.000000000 +0200
+++ ij/source/ij/gui/PlotWindow.java	2014-11-06 15:33:53.757278476 +0100
@@ -499,8 +499,12 @@
 	/** Draws a new plot in this window. */
 	public void drawPlot(Plot plot) {
 		this.plot = plot;
-		if (imp!=null)
+		if (imp!=null) {
 			imp.setProcessor(null, plot.getProcessor());	
+			ImagePlus plotImp = plot.getImagePlus();
+			Calibration plotCal = plotImp.getCalibration();
+			imp.setCalibration(plotCal);
+		}
 	}
 	
 	/** Called once when ImageJ quits. */
